tests/fuzz/s2n_client_hello_recv_fuzz_test.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <errno.h>
```

Comment:
I know this was a copy/paste from our other fuzz test but I think we can remove some unneeded includes. Seems like we'd only need:

- stdint.h
- api/s2n.h
- stuffer/s2n_stuffer.h
- utils/s2n_safety.h

Might have missed a few.